src/transformers/pipelines/token_classification.py

```diff
@@ -206,18 +204,27 @@ def __call__(self, inputs: Union[str, List[str]], **kwargs):
 
                 entity = {
                     "word": word,
-                    "score": score[idx][label_idx].item(),
-                    "entity": self.model.config.id2label[label_idx],
+                    "score": score[idx],
+                    "is_subword": is_subword,
                     "index": idx,
                     "start": start_ind,
                     "end": end_ind,
                 }
 
-                if self.grouped_entities and self.ignore_subwords:
-                    entity["is_subword"] = is_subword
-
                 entities += [entity]
 
+            if self.subword_label_re_alignment:
```

Comment:
we are going to set the labels according to the strategy, if subword_label_re_alignment == false we will leave the labels as they were predicted